tests/conftest.py

```diff
@@ -68,18 +68,21 @@ class Capture(object):
     def __init__(self, capfd):
         self.capfd = capfd
         self.out = ""
+        self.err = ""
 
-    def _flush_stdout(self):
+    def _flush(self):
         sys.stdout.flush()
```

Comment:
These double flushes are somewhat surprising, I assume it's to work around bugs. Care to write a comment about it?